CONTRIBUTING.md

```diff
@@ -580,6 +580,25 @@ rather than via `tox`. If you have installed the development packages in your py
 `pip install -r requirements-dev.txt`, then `ruff` and `black` will be available and can be run from
 the command line. See [`tox.ini`](tox.ini) for how `tox` invokes them.
 
+## Building API docs locally
+
+If you have made changes to the API documentation, you can run the commands below
+to build documentation locally to review the html output. 
+The easiest and recommended way to build the documentation is to use tox. 
+Once you run any of these command, the output will be located at `docs/_build/html`
+
+1. using [**tox**](https://tox.readthedocs.io/en/latest/#):
+
+```
+tox -edocs
+```
+
+2. using [**sphinx-build**](https://www.sphinx-doc.org/en/master/man/sphinx-build.html):
```

Comment:
@ElePT why document this? People can always figure it out if they want. My concern with documenting it is that it makes it a "blessed" path, which somewhat implies that we're going to commit to this working. 

It's valuable to have only one single endorsed way to do builds. If others want to do their custom thing, go ahead, but it's on them to verify it works properly.

For example, right now the below command is not actually the same as what we run with Tox:

https://github.com/Qiskit/qiskit/blob/7e73fcf51e1d3557e0f291bb757248ba14d39ebe/tox.ini#L81

And it doesn't install the optional requirements:

https://github.com/Qiskit/qiskit/blob/7e73fcf51e1d3557e0f291bb757248ba14d39ebe/tox.ini#L79

Reply:
That is a really good point and really well sold :) I was originally ok with the idea of showing another path but hadn't thought about committing to it working. Your suggestion below about clearing the cache also covers the case I had in mind (which was tox getting stuck).